Tests/presentation-translators.lisp

```diff
@@ -0,0 +1,42 @@
+(cl:in-package #:clim-tests)
+
+(def-suite* :mcclim.presentation-translators
+  :in :mcclim)
+
+(test presentation-translators.smoke
+  (define-command-table pt.smoke-ct)
+  (let ((tr (define-presentation-translator pt-smoke-tr
+                (integer string pt.smoke-ct)
+                (object)
+              (format nil "~a" object)))
+        (all (find-presentation-translators 'integer 'string 'pt.smoke-ct)))
+    (is (member tr all))))
+
+(test presentation-translators.meta
+  (define-command-table pt.smoke-ct2)
+  (let ((tr (define-presentation-translator pt-smoke-tr2
+                ((or real string) string pt.smoke-ct2)
+                (object)
+              (etypecase object
+                (real (format nil "~a" object))
+                (string object))))
+        (all-1 (find-presentation-translators 'real 'string 'pt.smoke-ct2))
+        (all-2 (find-presentation-translators 'string 'string 'pt.smoke-ct2))
+        (all-3 (find-presentation-translators '(or string real) 'string 'pt.smoke-ct2))
+        (all-4 (find-presentation-translators '(or string integer) 'string 'pt.smoke-ct2))
+        (all-5 (find-presentation-translators '(and string (member "a" "b")) 'string 'pt.smoke-ct2))
+        (all-6 (find-presentation-translators '(member "a" "b") 'string 'pt.smoke-ct2))
+        (all-7 (find-presentation-translators '(member "dan" 3) 'string 'pt.smoke-ct2))
+        (non-1 (find-presentation-translators 'number 'string 'pt.smoke-ct2))
+        (non-2 (find-presentation-translators '(or string number) 'string 'pt.smoke-ct2))
+        (non-3 (find-presentation-translators '(member "dan" :foo) 'string 'pt.smoke-ct2)))
+    (is (member tr all-1))
+    (is (member tr all-2))
+    (is (member tr all-3))
+    (is (member tr all-4))
+    (is (member tr all-5))
+    (is (member tr all-6))
+    (is (member tr all-7))
```

Comment:
I'm not sure what "all-6 and all-7 should both pass," in the commit message means. Doesn't this assert that the translator is found?

Reply:
in the test `presentation-translators.meta` we have multiple cases. I've added tests which does not pass, all-6 case is
(member "a" "b") -> string, and there exists a translator (or real string) -> , and (member "a" "b") is a subtype of (or real string) -- simpilar in the case all-7 (in both cases the translator should be found). I've changed to commit message to elaborate a bit more.

Reply:
I see. My remark about expected failures applies here as well in that case.